src/components/ToggleSwitch/index.tsx

```diff
@@ -0,0 +1,210 @@
+import * as React from 'react';
+import {
+  StyleSheet,
+  Animated,
+  TouchableOpacity,
+  Easing,
+  StyleProp,
+  Text,
+  TextStyle,
+  Platform,
+  TouchableOpacityProps
+} from 'react-native';
+import dimen from '../../utils/dimen';
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    alignItems: 'center'
+  },
+  basicStyle: {
+    height: dimen.normalizeDimen(16),
+    width: dimen.normalizeDimen(16),
+    borderRadius: dimen.normalizeDimen(16 / 2),
+    backgroundColor: '#2C67BC'
+  },
+  eahcStyles: {
+    fontSize: dimen.normalizeDimen(12),
+    color: '#f4f3f4',
+    position: 'absolute',
+    top: dimen.normalizeDimen(2),
+    left: dimen.normalizeDimen(3)
+  },
+
+  eachStylesOff: {
+    fontSize: dimen.normalizeDimen(12),
+    color: '#f4f3f4',
+    position: 'absolute',
+    top: dimen.normalizeDimen(2),
+    right: dimen.normalizeDimen(3)
+  },
+  mainStyes: {
+    borderRadius: dimen.normalizeDimen(12),
+    backgroundColor: '#F5F5F5',
+    height: dimen.normalizeDimen(20),
+    width: dimen.normalizeDimen(42),
+    padding: dimen.normalizeDimen(2),
+    ...Platform.select({
+      ios: {
+        shadowColor: 'rgba(0,0,0,0.25)',
+        shadowOffset: {width: 0, height: 0},
+        shadowOpacity: dimen.normalizeDimen(1),
+        shadowRadius: dimen.normalizeDimen(1)
+      },
+      android: {
+        elevation: dimen.normalizeDimen(1)
+      }
+    })
+  },
+  labelLeft: {
+    marginRight: 5,
+    color: '#FFFFFF',
+    fontSize: 12,
```

Comment:
should be normalized too? @febrimaulana

Reply:
saya tidak tahu kenapa kalo di sini saya coba normalized dia akan jadi aneh pak
`labelLeft: {
    marginRight: dimen.normalizeDimen(5),
    color: '#FFFFFF',
    fontSize: dimen.normalizeDimen(12),
    fontWeight: '400'
  },
  labelRight: {
    marginLeft: dimen.normalizeDimen(5),
    color: '#FFFFFF',
    fontSize: dimen.normalizeDimen(12),
    fontWeight: '400'
  }`
  
  hasil di screen nya jadi seperti ini pada iphone 11 pro max
  
![IMG_4452](https://github.com/BetterSocial/mobileapp/assets/30101915/d5ca5c70-d26c-4d52-9b8b-0017bf841d71)

text menjadi sangat mepet dengan box container biru tersebut
cc @fajarism

Reply:
apakah masih perlu untuk di normalized pada bagian itu pak, jika iya hasil nya akan seperti gambar di atas